packages/serverless-orchestration/src/ServerlessSpoke.js

```diff
@@ -54,17 +57,28 @@ spoke.post("/", async (req, res) => {
     if (execResponse.error) {
       throw execResponse;
     }
+    // Log the full execResponse in this log. This enables you to retrieve the full output of the child process.
     logger.debug({
       at: "ServerlessSpoke",
       message: "Process exited with no error",
       childProcessIdentifier: _getChildProcessIdentifier(req),
       execResponse,
     });
-    await delay(waitForLogsS); // Wait a few seconds to be sure the the winston logs are processed upstream.
+    await waitForLogger(logger);
+    await delay(waitForLoggerDelay); // Wait a few seconds to be sure the the winston logs are processed upstream.
+
+    // Send back a redacted log to the hub, if possible. This is done to decrease the amount of logs that are sent to
+    // the hub. extract only the `message` filed from the logs. This is the main log headline without any details.
     res.status(200).send({
       message: "Process exited with no error",
       childProcessIdentifier: _getChildProcessIdentifier(req),
-      execResponse,
```

Comment:
logs returned to the hub are now mapped to contain only the `message`. if there is no `message (or there is not an array of logs) then we return the full execResonce's stdout. this would be the case if the spoke is running a bash file or something equivalent.